<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="btnEncode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Encode</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>621</width>
      <height>221</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowCode">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Codes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWrite">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblString">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Your String:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Output:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>621</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRead">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtFilePath">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPath">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File path:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnEncode</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Encode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnShowCode</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ShowCodes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnWrite</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>WriteFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRead</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ReadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Encode()</slot>
  <slot>ShowCodes()</slot>
  <slot>WriteFile()</slot>
  <slot>ReadFile()</slot>
 </slots>
</ui>
